Description: Translates the cancel button on the GDM login screen
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/gdm/+bug/757468
From fe49514e73ddbcaa122539184401a2e3e80d278c Mon Sep 17 00:00:00 2001
Author: Gabor Kelemen <kelemeng@gnome.hu>
Forwarded: not-needed

diff --git a/gui/simple-greeter/gdm-greeter-login-window.ui b/gui/simple-greeter/gdm-greeter-login-window.ui
index d8cf1cc..9f1bc69 100644
--- a/gui/simple-greeter/gdm-greeter-login-window.ui
+++ b/gui/simple-greeter/gdm-greeter-login-window.ui
@@ -236,7 +236,7 @@
                     <property name="layout_style">end</property>
                     <child>
                       <object class="GtkButton" id="cancel-button">
-                        <property name="label">Cancel</property>
+                        <property name="label" translatable="yes">Cancel</property>
                         <property name="visible">True</property>
                         <property name="can_focus">True</property>
                       </object>
--
cgit v0.9
